packages/dropdowns/src/elements/Dropdown/Dropdown.tsx

```diff
@@ -17,20 +17,44 @@ import { DropdownContext } from '../../utils/useDropdownContext';
 export const REMOVE_ITEM_STATE_TYPE = 'REMOVE_ITEM';
 
 export interface IDropdownProps {
+  /** Opens the dropdown */
   isOpen?: boolean;
+  /** Identifies the currently selected item */
   selectedItem?: any;
+  /** Identifies the currently selected items */
   selectedItems?: any[];
+  /** Highlights an element at a selected index */
   highlightedIndex?: number;
+  /** Sets the value of the input element*/
   inputValue?: string;
+  /**
+   * Handles item selection
+   *
+   * @param {any|null} selectedItem The selected item value
+   * @param {ControllerStateAndHelpers<any>} stateAndHelpers
+   * */
   onSelect?: (selectedItem: any | null, stateAndHelpers: ControllerStateAndHelpers<any>) => void;
+  /**
+   * Handles state change
+   *
+   * @param {StateChangeOptions<any>} options
+   * @param {ControllerStateAndHelpers<any>} stateAndHelpers
+   * */
   onStateChange?: (
     options: StateChangeOptions<any>,
     stateAndHelpers: ControllerStateAndHelpers<any>
   ) => void;
+  /**
+   * Handles input value change
```

Comment:
@austingreendev should documentation be added around preferring `onInputValueChange` for improved IME (input-method editor) support?

@mikaelagurney there's a bit on this under https://zendeskgarden.github.io/react-components/dropdowns/#autocomplete-usage, but not incredibly helpful for understanding the "why?" of the recommendation.